production_services/settings.py

```diff
@@ -47,8 +47,12 @@
     'rest_framework_swagger',
     'ensembl_production_db.apps.EnsemblProductionDbConfig',
     'ensembl_website.apps.EnsemblWebsiteConfig',
+<<<<<<< HEAD
```

Comment:
You didn't merge this before opening your PR

Reply:
Yes I've noticed, forgot to save the file... see here: https://github.com/Ensembl/ensembl-production-services/pull/64/commits/ce9c8e86e29117a4a5c3eeba552104dc6bac2278